### DISCUSSION TITLE
Batch transcoding with photoprism convert

### DISCUSSION THREAD

**Discussion by benjaminjunior** [photoprism/photoprism#4008] [Q&A] [answered]
Hi there. I'm running the last build (231128-f48ff16ef). The docker container is running inside a VM on my proxmox hypervisor. 
I'm still waiting for an Nvidia Tesla P4 8G to speed up things a bi,t but meanwhile, I'm using the command "photoprism convert" to batch transcode all video files to .avc on the sidecar via CPU. 

My point is: every time I start the command, it takes ages (about 15-20min) to locate the first unconverted video, and resuming conversion of files not yet in the sidecar.

1st Question: Is this common behaviour (taking so many time to resume)? My library has 45k photos and 5k videos, feeded to the docker container via NFS on my local network.

2nd question: is it possible to pass any king or arguments to  "photoprism convert" to target a specific directory first?

Thank you.

**Comment by graciousgrey** [comment, marked as answer]
Hey 👋 

15 minutes to locate the first untranscoded video seems a bit long. I am not sure but this might be related to your setup. Maybe @lastzero has an idea.

The convert command lets you specify a directory and also file extensions. You can run photoprism convert --help to see all available options.

**Reply by benjaminjunior** [reply]
That was very helpful, thank you @graciousgrey. tbh, tried "-help". Didn't work, so I threw the towel :)
